### PR TITLE
Feature/tring image

### PR DESCRIPTION
**Pull request by KanasindhuPalanisamy**
Added the checkBox Component
Added the test cases for checkBox component

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5777,6 +5792,17 @@ react-native-fast-image@^8.6.3:
   resolved "https://registry.yarnpkg.com/react-native-fast-image/-/react-native-fast-image-8.6.3.tgz#6edc3f9190092a909d636d93eecbcc54a8822255"
   integrity sha512-Sdw4ESidXCXOmQ9EcYguNY2swyoWmx53kym2zRsvi+VeFCHEdkO+WG1DK+6W81juot40bbfLNhkc63QnWtesNg==
 
+<<<<<<< HEAD
```
Comment (deepaktringapps) [inline yarn.lock:5795]: Can you remove merge conflicts line

File: yarn.lock [hunk 2]
```diff
@@ -6981,7 +7007,11 @@ yargs-parser@^20.2.2, yargs-parser@^20.2.3, yargs-parser@^20.2.9:
   resolved "https://registry.yarnpkg.com/yargs-parser/-/yargs-parser-20.2.9.tgz#2eb7dc3b0289718fc295f362753845c41a0c94ee"
   integrity sha512-y11nGElTIV+CT3Zv9t7VKl+Q3hTQoT9a1Qzezhhl6Rp21gJ/IVTW7Z3y9EWXhuUBC2Shnf+DX0antecpAwSP8w==
 
+<<<<<<< HEAD
```
Comment (deepaktringapps) [inline yarn.lock:7010]: Remove merge conflicts line

File: src/components/checkBox/checkBox.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        lineHeight: 18,
+    },
+    wrapper: {
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    spaceStyle: {
+        marginRight: 5,
+    },
+});
```
Comment (deepaktringapps) [inline src/components/checkBox/checkBox.tsx:102]: Keep extra line at end of the file

File: package.json [hunk 4]
```diff
@@ -67,6 +67,7 @@
     "prettier": "^2.0.5",
     "react-native-builder-bob": "^0.20.0",
     "react-native-fast-image": "^8.6.3",
+    "react-native-vector-icons": "^9.2.0",
```
Comment (deepaktringapps) [inline package.json:70]: If vector icon not used please remove

**Review by deepaktringapps — CHANGES_REQUESTED** [review]
Please update the comments and run the npm and verify working as expected

File: yarn.lock (hunk 1 above)
Comment (KanasindhuPalanisamy) [reply yarn.lock:5795]: Yeah sure

**Review by deepaktringapps — APPROVED** [review]

**Pull request merged**